tfjs-backend-webgpu/src/kernels/FromPixels.ts

```diff
@@ -63,10 +63,8 @@ export function fromPixels(args: {
       [pixels.width, pixels.height];
   const outputShape = [height, width, numChannels];
 
-  // Disable importExternalTexture temporarily as it has problem in spec and
-  // browser impl
   const importVideo =
-      false && env().getBool('WEBGPU_IMPORT_EXTERNAL_TEXTURE') && isVideo;
+      env().getBool('WEBGPU_IMPORT_EXTERNAL_TEXTURE') && isVideo;
```

Comment:
I remember we have a related test case. Do you need to change the corresponding test?

Reply:
Yes, I just added the fix to the test case. Thanks!